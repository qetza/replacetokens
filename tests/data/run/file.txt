0: #{0}#
1.var4: #{1.VAR4}#
var1: #{var1}#
var2: #{var2}#
var2.sub2.0: #{VAR2.sub2.0}#
var3: #{VAR3}#
var_yml1: #{VAR_yml1}#
var_yml2: #{var_YML2}#
var_yaml1: #{VAR_yaml1}#
var_yaml2: #{var_YAML2}#