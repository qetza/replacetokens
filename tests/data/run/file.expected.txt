0: array
1.var4: array
var1: value1
var2: file
var2.sub2.0: value2
var3: env
var_yml1: file
var_yml2: inline
var_yaml1: inline
var_yaml2: file